app/models/Frontend.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.mvc.PathBindable
+
+sealed trait Frontend {
+  val collectionName: String
+  val binder: String
+
+  override def toString: String = binder
+}
+
+object Frontend {
+
+  case object Hub extends Frontend {
+    override val collectionName: String = "manage-transit-movements-departure-frontend"
+    override val binder: String         = "hub"
```

Comment:
I would make the binder the same as the collection name therefore in kibana we can trace exactly the frontend in the uri call.